addons/point_of_sale/static/src/js/models.js

```diff
@@ -419,6 +419,13 @@ exports.PosModel = Backbone.Model.extend({
                 return new exports.Product({}, product);
             }));
         },
+    },{
+        model: 'account.cash.rounding',
+        fields: ['name', 'rounding'],
+        domain: function(self){return self.config_id.rounding_method;},
```

Comment:
don't do a function for domain, just a normal domain ['id', '=', self.config_id.rounding_method]